Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -124,6 +124,14 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)thread
                         actionBlock:^{
                             [DebugUIMessages sendFakeMessages:10 * 1000 thread:thread];
                         }],
+        [OWSTableItem itemWithTitle:@"Create 100k fake messages"
+                        actionBlock:^{
+                            [DebugUIMessages sendFakeMessages:100 * 1000 thread:thread];
+                        }],
+        [OWSTableItem itemWithTitle:@"Create 100k fake text messages"
+                        actionBlock:^{
+                            [DebugUIMessages sendFakeMessages:100 * 1000 thread:thread isTextOnly:YES];
```

Comment:
These are actually fast if you comment out any logging triggered by saving the messages.  I haven't included all of those changes in this PR since we probably want that logging in production.